src/views/History/index.jsx

```diff
@@ -11,6 +11,9 @@ function History() {
   async function handleFormSubmit(data) {
     const [err, history] = await tryCatch(getHistory(data));
 
+    // shut the linter up for now
```

Comment:
? 😅

Reply:
`eslint` was throwing some errors about unused variables for me, I just wanted to quiet it down :)

Reply:
lol. I think you can make it quiet via

```jsx
// eslint-disable-next-line no-console
console.log('...');
```